openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1133,3 +1137,20 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def match_volumes(self, prefix):
```

Comment:
This function is not specific to scale, move this function to baseclass.

Reply:
Yes, okay moving to baseclass